api/GPUDevice.json

```
@@ -155,6 +155,47 @@
             "deprecated": false
           }
         },
+        "descriptor_entries_option_accepts_GPUBuffer_resource": {
+          "__compat": {
+            "description": "Bind `GPUBuffer` in place of a `GPUBufferBinding`.",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "138",
+                "partial_implementation": true,
+                "notes": "Supported on ChromeOS, macOS, and Windows only."
+              },
+              "chrome_android": {
+                "version_added": "138"
+              },
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
         "descriptor_entries_option_accepts_GPUTextureView_resource": {
```

But then we should also update this one:

```suggestion
        "descriptor_entries_resource_parameter_accepts_GPUTextureView": {
```